android/modules/ui/src/java/ti/modules/titanium/ui/widget/tableview/TableViewHolder.java

```diff
@@ -228,7 +233,18 @@ public void bind(final TableViewRowProxy proxy, final boolean selected)
 
 		// Set font and text color for title.
 		TiUIHelper.styleText(this.title, properties.getKrollDict(TiC.PROPERTY_FONT));
-		this.title.setTextColor(TiConvert.toColor(properties.optString(TiC.PROPERTY_COLOR, "black")));
+
+		// Determine title color based on background.
+		final int tableBackgroundColor =
+			TiConvert.toColor(tableViewProxy.getProperties(), TiC.PROPERTY_BACKGROUND_COLOR);
+		final int rowBackgroundColor =
+			TiConvert.toColor(properties, TiC.PROPERTY_BACKGROUND_COLOR);
+		final int backgroundColor = rowBackgroundColor != Color.TRANSPARENT ? rowBackgroundColor : tableBackgroundColor;
+		final int defaultTitleColor = backgroundColor == Color.TRANSPARENT || backgroundColor < (Color.BLACK / 2)
```

Comment:
Does this work with semantic colors / dark mode? May need to validate prior to merging